tubesize/src/components/options.tsx

```
@@ -0,0 +1,132 @@
+                                await chrome.storage.sync.set({
+                                    apiFallback: isChecked,
+                                });
+                                setAPIFallback(isChecked);
+                            }}
+                        ></button>
+                        <div className="i  nfo-tooltip-content" role="tooltip">
+                            Uses our backup server when local extraction fails. It can be slower.
+                            See
+                            <a
+                                href="https://github.com/MohamedSayed0573/TubeSize_Extension/blob/main/PRIVACY.md"
+                                target="_blank"
```

Biome (2.4.6)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

(lint/security/noBlankTarget)